helm/cluster-api-ipam-provider-in-cluster/templates/custom/manager-netpol.yaml

```diff
@@ -31,11 +31,8 @@ spec:
     - ipBlock:
         cidr: {{ . }}
     {{- end }}
-  # deny ingress traffic
-  ingress: []
   policyTypes:
   - Egress
-  - Ingress
```

Comment:
as for this change, it's not obvious from the PR, but there is a net policy defined in the same file covering the ingress - https://github.com/giantswarm/cluster-api-ipam-provider-in-cluster-app/blob/main/helm/cluster-api-ipam-provider-in-cluster/templates/custom/manager-netpol.yaml#L41